hybrasyl/Login.cs

```diff
@@ -65,7 +65,7 @@ private void PacketHandler_0x02_CreateA(Client client, ClientPacket packet)
             // isn't valid.
             byte passwordErr = 0x0;
 
-            if (Game.World.PlayerExists(name))
+            if (Game.World.PlayerExists(name) || (Game.Config.Access?.IsReservedName(name) ?? false))

```

Comment:
would this be better as `if ((Game.World.PlayerExists(name) || (Game.Config.Access?.IsReservedName(name))  == true))` ?

Reply:
needs coalescing as `Access` isn't required xml element in config